pkg/ddevapp/schema.json

```
@@ -0,0 +1,401 @@
+{
+  "$id": "https://json.schemastore.org/base.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "DDEV 1.22 project config",
```

Wouldn't this be better as just "DDEV project config" ? Easier to maintain... 
```suggestion
  "title": "DDEV project config",
```